admin_manual/enterprise/firewall/file_firewall.rst

```diff
@@ -132,6 +132,29 @@ Block by System Tag::
 .. figure:: images/firewall-3.png
    :alt: Protecting files tagged with "Confidential" from outside access
 
+Logging
+~~~~~~~
+
+Firewall logging can be set to **Off**, **Blocked Requests Only** or **All Requests**
+
+Off
+^^^
+
+The firewall blocks according to the defined rules but does not log any of its actions.
+
+Blocked Requests Only
+^^^^^^^^^^^^^^^^^^^^^
+
+The firewall logs blocked requests to the system log at **warning** level. To see these logs, the system log level must be
+set to include **warning**.
+
+All Requests
+^^^^^^^^^^^^
+
+The firewall logs both blocked and succcessful requests to the system log at **info** level. To see these logs, the
+system log level must be set to include **info**. Logging All Requests can generate a large amount of log data. It is
```

Comment:
1. ``Logging All Requests`` --> ``Logging all requests`` (see blocked requests above)
2. I would put the "Logging all request ..." sentence as a note. This highlights the impact much better

Reply:
Agree